javascript/features/settings/settings.js

```diff
@@ -33,11 +33,6 @@ class Settings extends Feature {
         for (const setting of SettingList)
             this.settings_.set(setting.identifier, setting);
 
-        // Import player specific settings to also work server wide.
```

Comment:
Is this related to your change?

Reply:
Not really.

I updated the PR subscription. I'll make a seperate PR next time.
I came past this code thus took it with.